core/blockchain.go

```diff
@@ -1124,6 +1124,14 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, []
 	abort, results := bc.engine.VerifyHeaders(bc, headers, seals)
 	defer close(abort)
 
+	if bc.Config().IsMCIP0(common.Big0) {
```

Comment:
Similar to comment from `consensus/ethash/consensus.go`, I'm not trying to understand or improve this code; I just wanted the simplest way to replicate.